<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1941</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>48</y>
      <width>311</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Công việc</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Công việc sau</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời gian ca</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>SPHC</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>A,B,C</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>M,N</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>N,U</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>U,V</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>O,I</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>I,H</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>U</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>I,K</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>I,H,K,O,V</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Tck = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Đường đi</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Sơ đồ cây</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Sơ đồ thời gian</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>1581</width>
      <height>681</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1579</width>
       <height>679</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>SƠ ĐỒ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>740</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Copyright by Đạt Nguyễn Thành, Duong Hai, Quân Đậu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QUẢN LÝ CÔNG NGHIỆP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1941</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
